app/models/apple/episode.rb

```diff
@@ -388,6 +388,10 @@ def drafting?
       apple_json&.dig("attributes", "publishingState") == "DRAFTING"
     end
 
+    def archived?
+      apple_json&.dig("attributes", "publishingState") == "ARCHIVED"
```

Comment:
What does "archived" mean?  Is that some action someone took manually in apple connect?  Or an episode that fell out of the public RSS?  Or something else?